python/cuml/metrics/cluster/__init__.py

```diff
@@ -22,3 +22,5 @@
 from cuml.metrics.cluster.mutual_info_score import \
    cython_mutual_info_score as mutual_info_score
 from cuml.metrics.cluster.entropy import cython_entropy as entropy
+from cuml.metrics.cluster.silhouette_score import (
+    silhouette_score, silhouette_samples)
```

Comment:
Can we do one line per function imported?

Reply:
Sure! If we want to standardize on that, let's add it to linting as well. I'll open an issue.